pkg/control/v2/control_test.go

```diff
@@ -2,10 +2,12 @@
 // or more contributor license agreements. Licensed under the Elastic License 2.0;
 // you may not use this file except in compliance with the Elastic License 2.0.
 
-package v1_test
+package v2
```

Comment:
Not sure why this was named `v1_test`, probably just a copy-paste from another time.  Cleaning up.